CONTRIBUTING.md

```diff
@@ -78,6 +78,15 @@ npm run dev
 You should now have an Express server running at
 [localhost:5000](http://localhost:5000) :tada:
 
+NOTE: Make sure you have all dependencies/packages updated. Try installing all dependencies before you run `npm run dev` . 
+
+Use `npm update` to update all dependencies before running run `npm install`.
```

Comment:
This isn't a good idea unless you're trying to make a pull request to update dependencies. Otherwise you'll have a lot of unnecessary noise in your pull request.

Reply:
I still don't think this is a good idea. Can you explain why you had to do this in order to get `electronjs.org`'s dependencies installed?